packages/blob-reader/helper.js

```diff
@@ -63,40 +61,47 @@ function getPath(el) {
     );
   }
 
-  let ret = $(rootSelector)
-    .filter(function () {
-      return $(this).text().trim() === 'View file';
-    })
-    .attr('href');
+  const [link] = Array.from(rootSelector).filter((value) => {
+    return value.textContent.trim() === 'View file';
+  });
+
+  let ret = link && link.getAttribute('href');
 
   if (!ret) {
-    ret = $('.js-permalink-shortcut').attr('href');
+    const shortcut = document.querySelector('.js-permalink-shortcut');
+
+    ret = shortcut && shortcut.getAttribute('href');
   }
 
   // When current page is a gist, get path from blob name
   if (!ret && isGist()) {
-    ret = $('.gist-blob-name', el.parentElement).text().trim();
+    ret = el.parentElement.querySelector('.gist-blob-name').textContent;
```

Comment:
The nice thing about jQuery is the build in fault tolerance. If the selector is not matching anything the result of the operation on the left was `undefined` whereas on the right, it will throw an error `Cannot read property 'textContent' of null`

Again, optional chaining is here to help

```suggestion
    ret = el.parentElement.querySelector('.gist-blob-name')?.textContent;
```

Reply:
Do you mind adding optional chaining everywhere?

Reply:
Nope, I prefer it. I use it in TypeScript but keep forgetting it's supported in JS now too.